releases/v19.2.0-alpha.20190701.md

```diff
@@ -0,0 +1,155 @@
+---
+title: What&#39;s New in v19.2.0-alpha.20190701
+toc: true
+summary: Additions and changes in CockroachDB version v19.2.0-alpha.20190701 since version v19.2.0-alpha.20190606
+---
+
+## July 01, 2019
+
+This release contains enhancements and bug fixes to the 19.2.0 series, including:
+
+- An initial prototype of support for using `IMPORT INTO` to ingest CSV data into an existing table. Usage: `IMPORT INTO t(a, b) CSV DATA ('gs://acme-co/employees.csv')`.
```

Comment:
Might wanna change "An initial prototype of support" > "Added experimental support"

Reply:
Fixed.